### DISCUSSION TITLE
画质设定(defaultVideoQuality)功能没有了么

### DISCUSSION THREAD

**Discussion by hstdt** [the1812/Bilibili-Evolved#1974] [提问 / Q&A] [answered]
之前的版本能选4K，现在每次看视频都是自动，好麻烦

**Comment by the1812** [comment, marked as answer]
没了, v2.0 中可能会再回归

**Reply by hstdt** [reply]
收到，我还以为记错这个了。把老的配置复制过来就开启了。期待2.0的回归。

`"defaultVideoQuality": "4K",`

**Reply by Geeyun-JY3** [reply]
感觉 `默认视频画质` 可以照顾到的需求一个是 #1271（但以前那种实现方式不能覆盖，可能要加一个 fallback 机制什么的，即在无 60 帧时选择切换至当前分辨率或更高一级的分辨率）；另一个就是一般情况下只用某个清晰度看视频，不受某些特殊视频或场景下偶尔手动切换清晰度影响。

**Comment by RingingVolcano** [comment]
现在B站官方也支持默认画质记忆了吧

**Reply by hstdt** [reply]
我这边在Safari上失效了，不知道是不是测试版mac系统的原因

**Reply by RingingVolcano** [reply]
建议在Chrome上再试一试 可能是浏览器的问题

**Reply by Geeyun-JY3** [reply]
> emmm没啥其他的事情也可以把issue关了

这是 discussion，只能管理员锁定。

而且本 repo 用 discussion 好像也没想着锁定，用完即走就是了：#1297

**Reply by RingingVolcano** [reply]
> > emmm没啥其他的事情也可以把issue关了
> 
> 这是 discussion，只能管理员锁定。
> 
> 而且本 repo 用 discussion 好像也没想着锁定，用完即走就是了：#1297

忘了这是讨论了（

**Reply by Geeyun-JY3** [reply]
不过现在倒是可以打 tag 了，就是不知道 @the1812 怎么想

**Reply by 1970633640** [reply]
对于非会员，还希望有720p60优先级大于1080p30的功能
